views/navigation.njk

```diff
@@ -38,12 +38,28 @@
             </a>
           </li>
           {% if isAuthenticated %}
-          <li class="nav-action">
-            <a class="usa-button usa-button-white" href="/logout">Log out</a>
-          </li>
+            <li class="nav-action">
```

Comment:
does the `isUAA` make sense anymore given that it's no longer binary?

Reply:
I think we'll need it until we turn on the multi auth flag and possibly for staging until we move completely to UAA.

Reply:
Same as above